clang-tools-extra/clang-tidy/tool/check_alphabetical_order_test.py

```diff
@@ -0,0 +1,395 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# To run these tests:
+# python3 check_alphabetical_order_test.py -v
+
+import io
+import os
+import tempfile
+import unittest
+from contextlib import redirect_stderr
+from typing import cast
+import textwrap
+
+
+import check_alphabetical_order as _mod
+
+
+class TestAlphabeticalOrderCheck(unittest.TestCase):
+    def test_normalize_list_rst_sorts_rows(self):
```

Comment:
```suggestion
    def test_normalize_list_rst_sorts_rows(self) ->None:
```

Same in similar cases.

Reply:
Sorry for the mistakes I made, I've fixed them as requested. Thanks again for reviewing :)